back-end/apps/api/src/api.module.ts

```diff
@@ -71,6 +71,10 @@ export const config = ConfigModule.forRoot({
     SchedulerModule.register({ isGlobal: true }),
   ],
   providers: [
+    {
```

Comment:
Does this order need to be swapped? If it matters, I think IpThrottling should happen first, even if I said otherwise before :D

Reply:
Done